TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.CfgAfterIterativePerformProcedureTransformer.cs

```diff
@@ -39,14 +48,18 @@ private bool Callback(BasicBlock<Node, D> block, int incomingEdge, BasicBlock<No
                 //If an after iterative group has been found, we must break the edge pointing to the PERFORM
                 //and replace it with a new edge pointing to the first instruction of the group.
                 if (iterativeGroup != null)
-                {
+                {                    
                     //Remove incoming edge
                     block.SuccessorEdges.RemoveAt(removedIndex);
 
                     //Create an edge to the first block of the iterative group.
                     int entranceEdge = cfg.SuccessorEdges.Count;
                     cfg.SuccessorEdges.Add(iterativeGroup.Group.First.Value);
                     block.SuccessorEdges.Add(entranceEdge);
+                    if(block.Context != null)
+                    {
+                        ((MultiBranchContext)block.Context).ChangeSuccessor(Builder, iterativeGroup, iterativeGroup.Group.First.Value);
+                    }
```

Comment:
As discussed, it seems it's not required as we always have a trnasition block (without context). Maybe add an assertion and remove the code ?

Reply:
Yes

Reply:
done in 96ffab5